pkg/server/postgres/postgres.go

```diff
@@ -128,6 +147,7 @@ func (p *PostgresPlugin) SyncDatabase(db *atlas.Database, dsn string) error {
 	if err != nil {
 		return err
 	}
+	fmt.Println(sqldb.Ping())
```

Comment:
Do we need to keep this ? better to use log instead of this ?

Reply:
I will remove